bootstrap/kubeadm/types/upstreamv1beta4/zz_generated.conversion.go

```diff
@@ -405,6 +405,7 @@ func Convert_v1beta2_BootstrapTokenString_To_upstreamv1beta4_BootstrapTokenStrin
 }
 
 func autoConvert_upstreamv1beta4_ClusterConfiguration_To_v1beta2_ClusterConfiguration(in *ClusterConfiguration, out *v1beta2.ClusterConfiguration, s conversion.Scope) error {
+	// WARNING: in.TypeMeta requires manual conversion: does not exist in peer-type
```

Comment:
This diff is interesting. TypeMeta was never actually converted, but now conversion-gen is adding a comment that we need manual conversion :)